TM1py/Services/CellService.py

```diff
@@ -252,6 +252,74 @@ def get_value(self, cube_name: str, elements: Union[str, Iterable] = None, dimen
         cellset = dict(self.execute_mdx(mdx=mdx, sandbox_name=sandbox_name, **kwargs))
         return next(iter(cellset.values()))["Value"]
 
+    def get_values(self, cube_name: str, element_list: Union[str, Iterable] = None, dimensions: List[str] = None,
```

Comment:
maybe  rename `element_list` to something more generic like `elements` to not give the wrong impression. Because actually this will be a list of lists of elements.
In the docstring you can put a small note or an example for this value.